src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/packaging-tools.props

```
@@ -1,5 +1,10 @@
 <Project>
 
+  <PropertyGroup>
+    <SigningToolsDirectory Condition="'$(SigningToolsDirectory)' != ''">$(RepoRoot)signing\</SigningToolsDirectory>
```

This is the convention I follow:

```suggestion
    <SigningToolsDir Condition="'$(SigningToolsDir)' != ''">$(RepoRoot)signing\</SigningToolsDir>
```